.github/workflows/ci_workflows.yml

```
@@ -0,0 +1,85 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Is this suppose to be changed before merge?
```suggestion
      fail-fast: true
```

---

I would go with true, but in fact am OK with either.

---

(hmm, wrong browser....)

---

I would normally go with yes but actually the numpy dev job might cause the others to stop so maybe keep false until we discuss allowed failures etc?

---

Yeah, you are right, the OSX job got canceled.

---

since we already have multiple workflows, I'm getting on board of indeed separating the allowed failures one out. In a follow-up.